src/python/README.md

```diff
@@ -0,0 +1,53 @@
+# Python Scripts for Merge Conflict Analysis
+
+
+
+
+This directory contains Python scripts designed to facilitate the analysis of merge conflicts using various merge tools. The scripts allow users to recreate merges, analyze conflicts, and compare different merge algorithms' across the base, conflict, and programmer merge.
+
+
+
+
+## Scripts Overview
+
+
+
+
+- `diff3_analysis.py`: This script analyzes merge conflicts for two merge tools on a given conflict.
+
+- It is different than the src/python/replay_merge, as it performs a 3 way diff between the base, conflicting branches, and the programmer merge.
```

Comment:
Could it first call `replay_merge.py`, and then do its own unique work?  That could make it much smaller and simpler.  If that is impossible or undesirable, why?

Reply:
Updated the readme.

It might be possible to use replay_merge.py for the underlying process of recreating the merge. However, this still wouldn't have the programmer merge. Moreover, the diff3_analysis method keeps the conflict and repos to diff3 everything and outputs a .txt file. Also, both methods use different systems for organizing the files (mine makes its own cloned repo folders), which would take some time to change.

Honestly, I think the functionality is different enough that I would prefer to leave both as separate methods. It would probably be a significant time investment refactoring, and looking at how different the actual code is, I'm sure where to begin. Both methods work successfully, and I don't this is worth the extra effort.